src/Kreta/Bundle/VCSBundle/spec/Kreta/Bundle/VCSBundle/Controller/Webhook/GithubWebhookControllerSpec.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/**
+ * This file belongs to Kreta.
+ * The source code of application includes a LICENSE file
+ * with all information about license.
+ *
+ * @author benatespina <benatespina@gmail.com>
+ * @author gorkalaucirica <gorka.lauzirika@gmail.com>
+ */
+
+namespace spec\Kreta\Bundle\VCSBundle\Controller\Webhook;
+
+use Doctrine\ORM\EntityManager;
+use Kreta\Component\VCS\Model\Interfaces\CommitInterface;
+use Kreta\Component\VCS\Serializer\Interfaces\SerializerInterface;
+use Kreta\Component\VCS\WebhookStrategy\GithubWebhookStrategy;
+use PhpSpec\ObjectBehavior;
+use Prophecy\Argument;
+use Symfony\Bridge\Doctrine\RegistryInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Symfony\Component\HttpFoundation\Request;
+
+class GithubWebhookControllerSpec extends ObjectBehavior
+{
+    function it_is_initializable()
+    {
+        $this->shouldHaveType('Kreta\Bundle\VCSBundle\Controller\Webhook\GithubWebhookController');
+    }
+
+    function it_responds_to_webhook(Request $request, ContainerInterface $container, GithubWebhookStrategy $strategy,
```

Comment:
This is the correcto format for multiline arguments:
function it_responds_to_webhook(
    Request $request,
    ContainerInterface $container
   ....
)